cli/drivers/ValetDriver.php

```diff
@@ -179,23 +179,55 @@ protected function isActualFile($path)
 
     /**
      * Load server environment variables if available.
+     * This method supports both the vale plus and valet variables files
+     * Processes any '*' entries first, and then adds site-specific entries
      *
      * @param  string  $sitePath
      * @param  string  $siteName
      * @return void
      */
-    protected function loadServerEnvironmentVariables($sitePath, $siteName)
+    public function loadServerEnvironmentVariables($sitePath, $siteName)
     {
-        $varFilePath = $sitePath . '/.env.valet';
-        if (! file_exists($varFilePath)) {
-            return;
+        // include files
+        $variableFiles = [
+            'ValetPlus' => $sitePath . '/.env.valet',
+            'Valet'     => $sitePath . '/.valet-env.php'
+        ];
+
+        foreach ($variableFiles as $variableFile) {
+            $exists = false;
+            if (file_exists($variableFile)) {
+                $exists = true;
+                break;
+            }
         }
-        $variables = include $varFilePath;
-        if (! isset($variables[$siteName])) {
-            return;
+
+        // if we find no include files then return out of this method
+        if(!$exists) { return; }
+
+        // build our variables from include files
+        foreach ($variableFiles as $key => $variableFile) {
```

Comment:
You loop through `$variableFiles` but you don't know if both files exist.
I would open and use `.env.valet`, otherwise look for `.valet-env.php` and use that. I don't think people will use both at the same time, since you should be able to do everything in `.valet-env.php` what you could do in `.env.valet` right? But the check on `.env.valet` prevents current implementations from breaking.

Reply:
Your'e right! What do you think of my change? I make sure that if the variableFile does not exist, I evict it from the variableFiles array